### DISCUSSION TITLE
Meetup for MONAI contributors and users at MICCAI2024 ?

### DISCUSSION THREAD

**Discussion by Lucas-rbnt** [Project-MONAI/MONAI#8078] [General]
Hello everyone,

I was wondering if some users or contributors are planning to attend the MICCAI conference in Morocco in October or if MONAI already has some planned sessions as in previous conferences.
If so, and if anyone is interested, it could be nice to meet up for a while to get to know each other, discuss our work, and exchange ideas.

Have a good day, and maybe see you soon!

**Comment by aylward** [comment]
Hi Lucas,

We will be giving an invited talk on MONAI at the MICCAI tutorial "AI-based image segmentation and labeling with free open source software, 3D Slicer”.   Also, if you're interested, perhaps we could pick a day for MONAI folks to sit at the same table(s) during lunch to chat.  It will be very informal, but I'd be happy to join.   Open to other suggestions...

Thanks,
Stephen

**Comment by Lucas-rbnt** [comment]
Hi,

Okay, that's great!
From what I've seen, your talk is on Sunday, October 6, and I'll only be arriving in the evening and won't be able to attend. 
Otherwise, yes my idea was exactly that, to meet at a coffee break or lunchtime between MONAI folks to chat a little. 

Have a nice day,
Lucas.

**Comment by aylward** [comment]
Hi,

Would you be willing to pick a day and time for this and share that info here?   Perhaps at the event hold up a sheet of paper with MONAI on it to attract folks?

I am getting pulled in a few different directions during MICCAI, so I cannot promise that I will be available during any particular break/lunch, but it would be great to join you if I can.

Sorry for passing this back to you, but maybe others here are willing to help as well...

Thanks,
Stephen

**Comment by Lucas-rbnt** [comment]
Hi,

Yes no problem, we can start by saying for example the lunch break on Monday October 7th at 12:30.
If others come forward with different constraints, we can always reorganize it.

Oh yes, I understand that it can be complicated to come if you are very busy, but no problem, we can try to organize it in this GitHub discussion at first and hope that other people are available!


Have a nice day,
Lucas.
